### PR TITLE
MGMT-4345 Removed setOperatorStatus from OCS validations code

### PR DESCRIPTION
**Pull request by priyanka19-98**
Signed off by: Priyanka Jiandani <pjiandan@redhat.com>

### CODE REVIEW

File: internal/operators/ocs/validations.go [hunk 1]
```diff
@@ -74,11 +53,6 @@ func (o *ocsValidator) ValidateRequirements(cluster *models.Cluster) (api.Valida
 
 	if int64(len(hosts)) < o.OCSRequiredHosts {
 		status = "Insufficient hosts to deploy OCS. A minimum of 3 hosts is required to deploy OCS. "
-		err = setOperatorStatus(cluster, status)
-		if err != nil {
-			o.log.Error("Failed to set Operator status ", err)
-			return api.Failure, "Failed to set Operator status "
-		}
 		o.log.Info("Setting Operator Status ", status)
```
Comment (gamli75) [inline internal/operators/ocs/validations.go:56]: please change the log to: "Validate Requirements status:"

File: internal/operators/ocs/validations.go [hunk 2]
```diff
@@ -101,11 +75,6 @@ func (o *ocsValidator) ValidateRequirements(cluster *models.Cluster) (api.Valida
 	} else if len(hosts) < o.OCSMasterWorkerHosts { // not supporting OCS installation for 2 Workers and 3 Masters
 		status = "Not supporting OCS Installation for 3 Masters and 2 Workers"
 		o.log.Info("Setting Operator Status", status)
```
Comment (gamli75) [inline internal/operators/ocs/validations.go:77]: please change the log to: "Validate Requirements status:"

File: internal/operators/ocs/validations.go [hunk 3]
```diff
@@ -116,10 +85,6 @@ func (o *ocsValidator) ValidateRequirements(cluster *models.Cluster) (api.Valida
 			if host.Role == models.HostRoleAutoAssign {
 				status = "All host roles must be assigned to enable OCS."
 				o.log.Info("Setting Operator Status ", status)
```
Comment (gamli75) [inline internal/operators/ocs/validations.go:87]: please change the log to: "Validate Requirements status:"

File: internal/operators/ocs/validations.go (hunk 1 above)
Comment (priyanka19-98) [reply internal/operators/ocs/validations.go]: fixed

File: internal/operators/ocs/validations.go (hunk 2 above)
Comment (priyanka19-98) [reply internal/operators/ocs/validations.go]: fixed

File: internal/operators/ocs/validations.go (hunk 3 above)
Comment (priyanka19-98) [reply internal/operators/ocs/validations.go]: fixed

**Review by gamli75 — APPROVED** [review]

**Review by gamli75 — APPROVED** [review]

**Pull request merged**